CONTRIBUTING.md

```diff
@@ -122,7 +122,7 @@ It can often be useful to publish a change and test it before doing a real live
 option for this.
 
 1. Check out pr branch
-2. `npx lerna publish --canary` - Publishing with canary generates a unique new version number, publishes it to npm, then updates the `canary` tag in npm to point to that new version.
+2. `npx lerna publish --canary --preid canary` - Publishing with canary generates a unique new version number, publishes it to npm, then updates the `canary` tag in npm to point to that new version.
```

Comment:
This command is starting to become a bit of a mouthful - do we want to create a package.json `scripts` shorthand for this? something like `canary`, so devs can just type `npm run canary` or `yarn canary` instead?

Reply:
Yes, that is a great idea. I can do that in a follow up PR.